base16-manager

```diff
@@ -465,43 +525,75 @@ set_random() {
   set_theme "$(list_themes -R | head -n1 | awk "{print $1;}")"
 }
 
+
+set_rofi() {
+  local package=$1
+  local theme=$2
+
+  # get template and handling for inexisting templates
+  if ls "$DATA_PATH/$package"/*/"base16-$theme."* &>/dev/null; then
+    file=$(ls "$DATA_PATH/$package"/*/"base16-$theme."*"rasi" )
+  else
+    err "$package: theme $theme not found."
+    return 1
+  fi
+
+  # path to rofi config
+  local ROFI_CONFIG="${XDG_CONFIG_HOME:-$HOME/.config}/rofi/config"
+
+  # remove old theme or new rasi style theme from config
+  sed -i -e '/^\! Base16 /d' \
+    -e '/^\! Author:/d' \
+    -e '/^\! base[[:xdigit:]]\{2\}/d' \
+    -e '/^\! Enable the extended coloring options/d' \
+    -e '/^rofi\.color-enabled: true/d' \
+    -e '/^\! Property Name/d' \
+    -e '/^rofi\.color-/d' \
+    -e '/^rofi\.theme/d' \
+    -e '/^# Theme set by base16-manager/d' \
+    -e '/^\s*theme:/d' "$ROFI_CONFIG"
```

Comment:
The lines
```
! Set the desired separator style
rofi.separator-style: solid
```
were not removed in my configuration. Is this on purpose?

Reply:
IMHO it is. Separator style is not theme specific. I would like to have same separator as I configured no matter to which theme I switch.

Reply:
I don't notice anything, after editing this line.

I think, the separator style is changed in the themes. I change the separator with
```
listview {
...
    border:           2px solid 0px 0px ;
....
}
```
compared to
```
listview {
...
    border:           2px dash 0px 0px ;
....
}

Reply:
you are right, that there is `border` rule in *rasi* themes, I probably didn't have it in my configs because they cleaned up well.

I'm not sure now how my proposed change works and I'm unable to check it at this moment, anyway it would be great if first time it found old-style b16m config it would replace it with new one and later it would replace only *rasi* theme content. That way I can make modifications which overwrite theme, like border style.

I'm writing all of this without checking the code.

Reply:
Good point. as the two (old b16m) lines  
```
! Set the desired separator style
rofi.separator-style: solid
```
are not contained in the rasi files, they can easily be removed with the other staff in `sed -e `.
The rest of my config was removed well.

Reply:
I still don't like the idea of removing lines which we might not add :) but I think that in current state, this approach is ok. 

Looks like old-style color border config is not overwriting rasi theme as I originally thought, so its bets as you suggested. Just pushed new version, tested on my different configs and seams ok.